### PR TITLE
Handle not validation errors as notifications

### PR DESCRIPTION
**Pull request by gabaldon**
This PR handle not validation errors as notifications.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -23,9 +23,13 @@ export default {
   },
   methods: {
     pollData() {
-      this.$store.dispatch('getBalance')
-      this.$store.dispatch('getTransactions', { limit: 50, page: 0 })
-      this.$store.dispatch('getAddresses')
+      const currentRoute = this.$router.currentRoute.path
+      const matchRoute = currentRoute.includes('/welcome-back' || '/ftu')
```
Comment (tmpolaczyk) [inline src/views/Home.vue]: I had to look it up, `'/welcome-back' || '/ftu'` evaluates to `'/welcome-back'`, thanks javascript.
```suggestion
      const matchRoute = currentRoute.includes('/welcome-back')  || currentRoute.includes('/ftu')
```
Also, shouldn't this use `.startsWith()` or similar?

File: src/utils.js [hunk 2]
```diff
@@ -2,6 +2,37 @@ import cbor from 'cbor'
...
+    }
+    case 'default': {
+      Notification.requestPermission(permission => {
+        if (permission === 'granted') {
+          return innerCreateNotification
+        }
+        alert('Notification denied')
+      })
+    }
+  }
```
Comment (Tommytrg) [inline src/utils.js]:
```suggestion
export function createNotification() {
  if (window.Notification) {
    Notification.requestPermission(permission => innerCreateNotification)
    } else {
      console.log('notifications not supported')
    }
  }
```

File: src/utils.js [hunk 3]
```diff
@@ -2,6 +2,37 @@ import cbor from 'cbor'
 import uuidv4 from 'uuid/v4'
 import { WIT_UNIT } from '@/constants'
 
+export function createNotification() {
```
Comment (Tommytrg) [inline src/utils.js]: Can you add a comment to this function?

File: src/utils.js [hunk 4]
```diff
@@ -2,6 +2,37 @@ import cbor from 'cbor'
...
+    case 'default': {
+      Notification.requestPermission(permission => {
+        if (permission === 'granted') {
+          return innerCreateNotification
+        }
+        alert('Notification denied')
+      })
+    }
+  }
+  function innerCreateNotification(notificationProps) {
```
Comment (Tommytrg) [inline src/utils.js]: Can you comment this function?

**Pull request merged**